src/shared/generate-syscall-list.py

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env python
 import sys
+import os
+
+s390 = 's390' in os.uname().machine
+arm = 'arm' in os.uname().machine
 
 for line in open(sys.argv[1]):
+    if line.startswith('s390_') and not s390:
+        continue
+    if line.startswith('arm_') and not arm:
+        continue
```

Comment:
i hope they never come up with a syscall where they us the verb "to arm"... i.e. "arm_timer()" or so ;-)